gc/gc-tool/src/main/java/org/projectnessie/gc/tool/cli/options/JdbcOptions.java

```diff
@@ -23,6 +23,10 @@
 import picocli.CommandLine;
 
 public class JdbcOptions {
+  @CommandLine.Option(
+      names = "--jdbc",
+      description = "Flag whether to use the JDBC contents storage.")
+  boolean jdbc;
 
   @CommandLine.Option(
       names = "--jdbc-url",
```

Comment:
I wonder if adding am explicit default value attribute of `${NESSIE_GC_JDBC_URL}` is going to make this option "set" :thinking:

Reply:
cf. https://github.com/projectnessie/nessie/blob/main/tools/content-generator/src/main/java/org/projectnessie/tools/contentgenerator/cli/CommittingCommand.java#L26

Reply:
The culprit is that it actually has the values via the defaults from the environment.